l10n_vn_account_invoice_number/views/account_invoice_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Invoices -->
+    <record id="view_invoice_supplier_form_inherit" model="ir.ui.view">
+        <field name="name">view.invoice.supplier.form.inherit</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="arch" type="xml">
+            <field name="reference" position="after">
+                <field name="invoice_number"/>
```

Comment:
For all views, hide those 3 fields when the related company_id is not in Vietnam, this will be useful in the case of cross-country multicompany.